gson/src/main/java/com/google/gson/JsonObject.java

```diff
@@ -155,6 +155,15 @@ public int size() {
     return members.size();
   }
 
+  /**
+   * Returns true if the number of key/value pairs in the object is zero.
+   *
+   * @return true if the number of key/value pairs in the object isgit sd zero.
+   */
```

Comment:
Could you please also add `@since` (see #2212):
```suggestion
   * @since $next-version$
   */
```

Reply:
Done.